crates/ruff_python_ast/src/visitor.rs

```diff
@@ -572,13 +572,7 @@ pub fn walk_expr<'a, V: Visitor<'a> + ?Sized>(visitor: &mut V, expr: &'a Expr) {
     }
 }
 
-pub fn walk_constant<'a, V: Visitor<'a> + ?Sized>(visitor: &mut V, constant: &'a Constant) {
-    if let Constant::Tuple(constants) = constant {
-        for constant in constants {
-            visitor.visit_constant(constant);
-        }
-    }
-}
+pub fn walk_constant<'a, V: Visitor<'a> + ?Sized>(visitor: &mut V, constant: &'a Constant) {}
```

Comment:
Should we remove all visiting of constants? As far as I can tell https://github.com/astral-sh/ruff/blob/22bcd0afb428a4ef4a9426ae6374693aa9bf8043/crates/ruff_python_ast/src/visitor.rs#L510 is unused now

Reply:
See #5840